### PR TITLE
Changes Cursed Katana to Chicken Suit

### PR DESCRIPTION
**Pull request by Groudonmaster**
Refer to issue #850 

Removes Cursed Katana from spawnable list

Gotta make a changelog will commit soon.

### CODE REVIEW

File: html/changelogs/example.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #################################
 
 # Your name.  
-author: N3X15
+author: Groudonmaster
```
Comment (AsV9) [inline html/changelogs/example.yml]: THAT'S NOT HOW CHANGELOGS WORK

Comment (Groudonmaster) [inline html/changelogs/example.yml]: FUK

**Comment by george99g** [comment]
Squash your commits, damn it.

**Comment by Groudonmaster** [comment]
I don't even know what that means George

**Comment by oisin100** [comment]
> Squash your commits, damn it.

Shhhh george... He hasn't got that many commits....

PS: You reverted your fuck-up but you havent added a new changelog.

**Comment by Groudonmaster** [comment]
I KNOWWW

**Comment by Cruix** [comment]
Don't forget to recompile the git stash merger with debug undefined so it doesn't combobulate when you squash the commits.

**Comment by Groudonmaster** [comment]
Ahhhh. I do what I want.

**Comment by FluffySurvivor** [comment]
but why

a katana is more badass than a chicken suit

**Comment by Groudonmaster** [comment]
Because it's easier than rewriting the code for cursed katana

**Comment by Groudonmaster** [comment]
Also the normal katana can still drop

**Comment by FluffySurvivor** [comment]
So if something is bugged, better remove it. 
10/10 good meme groudon

**Comment by DavidEdwards** [comment]
**honk**changelog**honk**

**Comment by Groudonmaster** [comment]
So if something is bugged, better remove it. 
10/10 good meme groudon

The cursed katana can still occur as a cursed weapon :p just can't from a box that it doesn't really belong in anyways

**Pull request merged**